### DISCUSSION TITLE
Can't watch in 720p60?

### DISCUSSION THREAD

**Discussion by kamild1996** [streamlink/streamlink-twitch-gui#1003] [Q&A] [answered]
When the quality is set to "Source", it is indeed playing in source quality.
When the quality is set to "High", it's playing 720p30.
But there's nothing in-between to allow me to watch in 720p60. Is there a way to get this quality option with streamlink-twitch-gui? Or is this a streamlink limitation?

**Comment by bastimeyer** [comment, marked as answer]
You need to reconfigure your stream quality name mapping:
- https://github.com/streamlink/streamlink-twitch-gui/wiki/Streamlink-configuration#stream-qualities
- https://streamlink.github.io/cli.html#cmdoption-stream-sorting-excludes

In regards to any limitations, read this:
- https://github.com/streamlink/streamlink/issues/4902
- https://github.com/streamlink/streamlink/issues/5764

**Reply by kamild1996** [reply]
Thank you, remapped "High" to 720p60 and it works :)
